core/unbound/start.py

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/python
+import os
+
+os.execv("/usr/sbin/unbound", [''])
```

Comment:
Couldn't this just be `CMD /usr/sbin/unbound` in the Dockerfile?

Then you could also remove python from the image

Reply:
yeah possible but maybe we want to change some settings via env vars later, so I included it exactly like in the other images.

Reply:
If I should remove it, let me know, I will change it quickly

Reply:
I think it would be better to include it when it's needed… without python the imagesize goes down to 10 MB (or 9 MB if you remove curl after using it :D)

Reply:
done :+1: